card_db_src/translation.md

```diff
@@ -0,0 +1,180 @@
+# Translation Instructions
+
+## File Format
+    /card_db/
```

Comment:
👍  I like the clarity across this whole file

Reply:
I actually didn't write any of this - @wvoigt did back when, I just moved files around.